### PR TITLE
Frontend: Add unit tests for update profile & permissions controller

### PR DESCRIPTION
**Pull request by Sanji515**
PR contains the front-end unit tests of :

- Update Profile Controller
- Permission Controller

### CODE REVIEW

**Comment by Sanji515** [comment]
@RishabhJain2018 @galipremsagar Please review

File: frontend/tests/controllers-test/permissionCtrl.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            return $controller('PermCtrl', {$scope: $scope});
+        };
+    }));
+
+    describe('Global Variables', function () {
+        beforeEach(function () {
+            spyOn(utilities, 'getData');
+        });
+
+        it('has default values', function () {
```
[the remark this answers is not in the document]
Comment (Sanji515) [reply frontend/tests/controllers-test/permissionCtrl.test.js]: done

File: frontend/tests/controllers-test/updateProfileCtrl.test.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+            expect(vm.stopLoader).toHaveBeenCalled();
+        });
+
+        it('when affiliation is invalid', function () {
+            var resetconfirmFormValid = true;
+            tryClauseResponse = affiliationInvalid;
+            success = false;
+
+            vm.updateProfile(resetconfirmFormValid);
+            expect(vm.startLoader).toHaveBeenCalledWith("Updating Your Profile");
```
[the remark this answers is not in the document]
Comment (Sanji515) [reply frontend/tests/controllers-test/updateProfileCtrl.test.js]: done

**Comment by galipremsagar** [comment]
@RishabhJain2018  This can be merged.

**Pull request merged**